### ISSUE TITLE
Fail to loads partition every time I redeploy milvus server

### ISSUE THREAD

**Issue by bishnusahu** [milvus-io/pymilvus#389]
The milvus server shows the following error in insert API every time I redeploy it:-

`milvus.client.exceptions.BaseException: <BaseException: (code=1, message=Fail to get partition )>`

Following is my code snippet for insert Api:- 

```
@app.route("/" + config["service_name"] + "/" + config["service_version"] + "/insert", methods=['POST'])
def insert():
    json_response = {}
    vectorWithIds = request.get_json()["vectors"]
    entities = []
    ids = []
    embeddings = []
    for vectorWithId in vectorWithIds:
        ids.append(vectorWithId["id"])
        embeddings.append(vectorWithId["vector"])
    entities.append({"name": "embedding", "values": embeddings, "type": DataType.FLOAT_VECTOR})
    ids = milvus.insert(MILVUS_COLLECTION_NAME, entities, ids)
```

As per the docs mentioned here (https://milvus.io/docs/v0.11.0/insert_delete_entity.md#Python) partition is not required to insert into milvus. I have verified the metadata in mysql database and the partition do exists there. If i create a new partition every time I redeploy, my search is coming up empty even if I am not specifying any partitions parameter for search.

Specs:-
Client Version :- 0.3.0
Config Version:- 0.6
Milvus Version:- 0.11.0

This is more of a sub-problem for me with the broader problem being every time the milvus server deploys, it pre-loads the collections but it doesn't preloads the partitions so the vectors inserted before are not searchable after deployment. Following is the code snippet for search api:-

```
@app.route("/" + config["service_name"] + "/" + config["service_version"] + "/search", methods=['GET'])
def search():
    json_response = {}
    vectorStr = flask.request.args.get('vector')
    topK = int(flask.request.args.get('topK'))
    vector = json.loads(vectorStr)
    query_hybrid = {
        "bool": {
            "must": [
                {
                    "vector": {
                        "embedding": {"topk": topK, "query": [vector], "metric_type": "L2"}
                    }
                }
            ]
        }
    }
    results = milvus.search(MILVUS_COLLECTION_NAME, query_hybrid, fields=["embedding"])
```

Steps to reproduce this issue:-
1. Start milvus server 0.11.0
2. Insert a vector in any partition.
3. restart the server
4. Search for the inserted vector

Please let me know if there are some configuration I am missing or any other help would be appreciated.

**Comment by BossZou** [comment]
@bishnusahu , I think you need invoke `insert()` after inserted. So the snippet for insert Api may be like:
```
@app.route("/" + config["service_name"] + "/" + config["service_version"] + "/insert", methods=['POST'])
def insert():
    json_response = {}
    vectorWithIds = request.get_json()["vectors"]
    entities = []
    ids = []
    embeddings = []
    for vectorWithId in vectorWithIds:
        ids.append(vectorWithId["id"])
        embeddings.append(vectorWithId["vector"])
    entities.append({"name": "embedding", "values": embeddings, "type": DataType.FLOAT_VECTOR})
    ids = milvus.insert(MILVUS_COLLECTION_NAME, entities, ids)
    milvus.flush([MILVUS_COLLECTION_NAME])
```

**Comment by bishnusahu** [comment]
@BossZou Thanks for replying. I have tried flushing also. And my auto_flust_interval is 1 second. But still no effect. 
As far as I understand, the issue is I am missing some configuration which allows a newly started milvus server to read already existing collections and partitions from mysql and enable search in them. For collections I have enabled the configuration `preload_collection` but that does not seems to be connecting to any existing partitions. Not even with `_default` partition.
